test/unittests/Microsoft.ServiceFabric.Services.Remoting.Tests/Microsoft.ServiceFabric.Services.Remoting.Tests.csproj

```diff
@@ -20,4 +21,8 @@
   <ItemGroup>
     <ProjectReference Include="$(RepoRoot)src\netstandard\Microsoft.ServiceFabric.Services.Remoting\Microsoft.ServiceFabric.Services.Remoting_netstandard.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <!-- BinarryFormatter compatibility package -->
```

Comment:
Could you check spelling? It would be helpful to include a link to the page explaining it.